### PR TITLE
LPS-93139 buildRest

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v1_0/DataDefinitionResourceImpl.java [hunk 1]
```diff
@@ -109,18 +111,8 @@ public DataDefinition getDataDefinition(Long dataDefinitionId)
 			Long siteId, String keywords, Pagination pagination)
 		throws Exception {
 
-		if (Validator.isNull(keywords)) {
```
Comment (jeyvison) [inline modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v1_0/DataDefinitionResourceImpl.java]: Should we do the same to data record collection?

**Comment by jeyvison** [comment]
Pull request submitted to https://github.com/marcelabc/liferay-portal/pull/125. [See changes here.](https://github.com/jeyvison/liferay-portal/compare/185e263ecd1f~3...185e263ecd1f) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**